### PR TITLE
Update initialization description and check for ro mode in drivers

### PR DESCRIPTION
**Pull request by LeonidVas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -302,14 +302,28 @@ build and install.
 
 # Using the `queue` module
 
+## Initialization
```
Comment (Totktonada) [inline README.md:305]: NB: I would also update ToC.

File: README.md [hunk 2]
```diff
@@ -302,14 +302,28 @@ build and install.
...
 ```lua
 queue = require 'queue'
 ```
 
 The request "require 'queue'" causes automatic creation of
 the `_queue` space, unless it already exists. The same request
 also sets the necessary space triggers and other objects
-associated with queues.
+associated with queues.<br />
```
Comment (Totktonada) [inline README.md:313]: Just splitting of spaces with an empty line should work for GFM (any markdown dialect in fact).

File: README.md [hunk 3]
```diff
@@ -302,14 +302,28 @@ build and install.
...
+If the instance hasn't been configured yet (`box.cfg()` hasn't been called),
+the initialization of the queue module will be deffered until the instance will
```
Comment (Totktonada) [inline README.md:315]: Typo: deffered -> deferred.

File: README.md [hunk 4]
```diff
@@ -302,14 +302,28 @@ build and install.
...
+be configured ("lazy start"). For a good work of the queue, it's necessary to
+run the instance in rw mode. If the instance run in ro mode, the initialization
+of the queue will be deffered until the instance will be configured in rw mode.
```
Comment (Totktonada) [inline README.md:318]: Same here.

**Review by Totktonada — APPROVED** [review]
LGTM.

File: README.md (hunk 2 above)
Comment (LeonidVas) [reply README.md]: Updated.
https://gist.github.com/shaunlebron/746476e6e7a4d698b373

File: README.md (hunk 1 above)
Comment (LeonidVas) [reply README.md:306]: Updated

File: README.md (hunk 3 above)
Comment (LeonidVas) [reply README.md]: Fixed

File: README.md (hunk 4 above)
Comment (LeonidVas) [reply README.md]: Fixed

**Pull request merged**